modules/ROOT/pages/redelivery-policy.adoc

```diff
@@ -0,0 +1,63 @@
+= Redelivery Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, flow, redelivery, policy
+
+A redelivery policy is a filter added to Mule sources, such as modules and connectors, to prevent the dispatch of messages known to cause errors.
```

Comment:
How can we involve your audience in this opening paragraph? It's great to tell what RP is, but WHY does your audience care ("A redelivery policy is a filter...that helps you avoid errors by preventing error-causing messages from being dispatched." Arg...even this uses passive voice.... "...by preventing <what> from dispatching error-causing messages")

Also, small consideration, but try to refer to the full name Mule runtime engine at the start of new topics before moving on to using Mule alone?